primefaces/src/main/java/org/primefaces/util/FileUploadUtils.java

```diff
@@ -69,6 +69,12 @@ public class FileUploadUtils {
 
     private static final Logger LOGGER = Logger.getLogger(FileUploadUtils.class.getName());
 
+    // Message keys
+    private static final String FILENAME_INVALID_NULL = "primefaces.FileValidator.FILENAME_NOT_NULL";
```

Comment:
either add _KEY or remove _KEY from others
i prefer without _KEY

also its currenlty inverted
FILENAME_NOT_NULL, FILENAME_INVALID_CHAR

i would just call it FILENAME_EMPTY

Reply:
good catch I have fixed them